al_aws_collector.js

```diff
@@ -120,23 +121,23 @@ class AlAwsCollector {
                 } else {
                     var endpoints = {
                         ingest_api : mapResult[0].ingest,
-                        azcollect : mapResult[1].azcollect
+                        azollect_api : mapResult[1].azcollect
```

Comment:
regarding this comment - https://github.com/alertlogic/al-aws-collector-js/pull/6#discussion_r191419205 - it turns that there is defect with current updateEndpoints code. 

I fixed it by chaning to `azOllect_api` (typo to be sorted later) in `updateEndpoints` method